core/env/each_key_spec.rb

```diff
@@ -18,8 +18,12 @@
     end
   end
 
-  it "returns an Enumerator if called without a block" do
-    ENV.each_key.should be_an_instance_of(Enumerator)
+  it "returns an Enumerator of the keys if called without a block" do
+    enum = ENV.each_key
+    enum.should be_an_instance_of(Enumerator)
+    enum.each do |key|
+      ENV.should include(key)
+    end
```

Comment:
How about:
```ruby
enum.to_a.should == ENV.keys
```
?

Reply:
Fine with me.  I was just thinking to iterate over the enum, rather than transform it.

Reply:
Enumerable#to_a iterates over the enum, so it should achieve the same, but the spec with `.to_a` is more concise and precise (it tests they are equal, not one include all of the other but not the reverse)